README.md

```diff
@@ -1 +1,452 @@
+   62  gzip -d data.gz
+   63  file data
+   64  mv data data.tar
+   65  tar xvf data.tar
+   66  ls
+   67  file data5.bin
+   68  mv data5.bin data6.tar
+   69  tar xvf data6.tar
+   70  file data6.bin
+   71  mv data6.bin data7.bz
+   72  bzip -d data7.bz
+   73  bzip2 -d data7.bz
+   74  ls
+   75  file data7
+   76  mv data7 data7.tar
+   77  tar xvf data7.tar
+   78  file data8.bin
+   79  mv data8.bin data9.gz
+   80  gzip -d data9.gz
+   81  ls
+   82  fiel data9
+   83  file data9
+   84  cat data9
+   85  history
+the password for the fourteenth level is  fGrHPx402xGC7U7rXKDaxiWFTOiF0ENq and the commmand used is ssh bandit14@localhost -i sshkey.private and then cat /etc/bandit_pass/bandit14
+the password for the fifteenth level is jN2kgmIXJ6fShzhT2avhotn4Zcka6tnt and the command used is nc localhost 30000 and then fGrHPx402xGC7U7rXKDaxiWFTOiF0ENq
+the password for the sixteenth level is JQttfApK4SeyHwDlI9SXGR50qclOAil1 and the command used is echo jN2kgmIXJ6fShzhT2avhotn4Zcka6tnt | openssl s_client -connect localhost:30001 -ign_eof
+the password for the seventeenth level is not given. it's credentials are given to us. 
+these are 
+-----BEGIN RSA PRIVATE KEY-----
+MIIEogIBAAKCAQEAvmOkuifmMg6HL2YPIOjon6iWfbp7c3jx34YkYWqUH57SUdyJ
+imZzeyGC0gtZPGujUSxiJSWI/oTqexh+cAMTSMlOJf7+BrJObArnxd9Y7YT2bRPQ
+Ja6Lzb558YW3FZl87ORiO+rW4LCDCNd2lUvLE/GL2GWyuKN0K5iCd5TbtJzEkQTu
+DSt2mcNn4rhAL+JFr56o4T6z8WWAW18BR6yGrMq7Q/kALHYW3OekePQAzL0VUYbW
+JGTi65CxbCnzc/w4+mqQyvmzpWtMAzJTzAzQxNbkR2MBGySxDLrjg0LWN6sK7wNX
+x0YVztz/zbIkPjfkU1jHS+9EbVNj+D1XFOJuaQIDAQABAoIBABagpxpM1aoLWfvD
+KHcj10nqcoBc4oE11aFYQwik7xfW+24pRNuDE6SFthOar69jp5RlLwD1NhPx3iBl
+J9nOM8OJ0VToum43UOS8YxF8WwhXriYGnc1sskbwpXOUDc9uX4+UESzH22P29ovd
+d8WErY0gPxun8pbJLmxkAtWNhpMvfe0050vk9TL5wqbu9AlbssgTcCXkMQnPw9nC
+YNN6DDP2lbcBrvgT9YCNL6C+ZKufD52yOQ9qOkwFTEQpjtF4uNtJom+asvlpmS8A
+vLY9r60wYSvmZhNqBUrj7lyCtXMIu1kkd4w7F77k+DjHoAXyxcUp1DGL51sOmama
++TOWWgECgYEA8JtPxP0GRJ+IQkX262jM3dEIkza8ky5moIwUqYdsx0NxHgRRhORT
+8c8hAuRBb2G82so8vUHk/fur85OEfc9TncnCY2crpoqsghifKLxrLgtT+qDpfZnx
+SatLdt8GfQ85yA7hnWWJ2MxF3NaeSDm75Lsm+tBbAiyc9P2jGRNtMSkCgYEAypHd
+HCctNi/FwjulhttFx/rHYKhLidZDFYeiE/v45bN4yFm8x7R/b0iE7KaszX+Exdvt
+SghaTdcG0Knyw1bpJVyusavPzpaJMjdJ6tcFhVAbAjm7enCIvGCSx+X3l5SiWg0A
+R57hJglezIiVjv3aGwHwvlZvtszK6zV6oXFAu0ECgYAbjo46T4hyP5tJi93V5HDi
+Ttiek7xRVxUl+iU7rWkGAXFpMLFteQEsRr7PJ/lemmEY5eTDAFMLy9FL2m9oQWCg
+R8VdwSk8r9FGLS+9aKcV5PI/WEKlwgXinB3OhYimtiG2Cg5JCqIZFHxD6MjEGOiu
+L8ktHMPvodBwNsSBULpG0QKBgBAplTfC1HOnWiMGOU3KPwYWt0O6CdTkmJOmL8Ni
+blh9elyZ9FsGxsgtRBXRsqXuz7wtsQAgLHxbdLq/ZJQ7YfzOKU4ZxEnabvXnvWkU
+YOdjHdSOoKvDQNWu6ucyLRAWFuISeXw9a/9p7ftpxm0TSgyvmfLF2MIAEwyzRqaM
+77pBAoGAMmjmIJdjp+Ez8duyn3ieo36yrttF5NSsJLAbxFpdlc1gvtGCWW+9Cq0b
+dxviW8+TFVEBl1O4f7HVm6EpTscdDxU+bCXWkfjuRb7Dy9GOtt9JPsX8MBTakzh3
+vBgsyi/sN3RqRBcGU40fOoZyfAMT8s1m/uYv52O6IgeuZ/ujbjY=
+-----END RSA PRIVATE KEY-----
+
+the commands used are 
+    1  clear
+    2  ls
+    3  clar
+    4  clear
+    5  nmap -p 31000-32000 localhost
```

Comment:
Explain the `nmap` commands used by you. (explain only these two commands that you have executed)

Reply:
I have used the nmap command here to a perform a scan of the active ports in the range of ports specified ie. 31000-32000. the -p flag is used to do that. 

I read the article https://linuxhint.com/nmap_version_scan/ and came across this line : 
"This feature will only show results if uncommon services are running on the target, there will not be differences in servers with widely used services"
So I thought that some hidden ports might be there as the default intensity is 7 but both the commands gave the same lists of ports.